index.html

```diff
@@ -5,11 +5,161 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+    <title>VCC Conference</title>
 </head>
 
 <body>
-    <h1>My heading</h1>
+    <header>
+        <h1>vcc</h1>
+        <div class="nav-bar">
+            <input id="toggle" class="menu-toggle" type="checkbox" aria-hidden="true">
+            <label for="toggle" class="nav-icon" aria-hidden="true">
+                <span class="icon-line"></span>
+                <span class="icon-line"></span>
+                <span class="icon-line"></span>
+            </label>
+            <nav class="nav">
+                <ul class="nav-items">
+                    <li class="nav-item"><a href="#home">Home</a></li>
+                    <li class="nav-item"><a href="#speakers">Speakers</a></li>
+                    <li class="nav-item"><a href="#schedule">Schedule</a></li>
+                </ul>
+            </nav>
+        </div>
+        <section id="home" class="home">
+            <h1>visual communication conference</h1>
+            <p class=date>pasadena, california </p>
+            <p class=date>may 4-6, 2019</p>
+            <div class="logo-container">
+                <div class="circle"></div>
+                <div class="circle"></div>
+            </div>
+            <p class="button"><a href="#" target="_blank">Buy Ticket</a></p>
+        </section>
+
+    </header>
+    <main>
+        <p class="discription">VCC (Visual Communication Conference) is the largest design conference about Visual
+            Communication.
+            We share our opinions, ideas together and try to find out better ways to communicate with people around the
+            world.
+            Experience the design trend through the world and meet with nearly 2,000 attendees from inside and outside
+            of
+            U.S.
+        </p>
+        <!-- https://www.howdesignlive.com/about-->
+        <section id="speakers" class="speakers">
+            <h2>Speakers</h2>
+            <ul class="clearfix">
+
+                <figure class="adam"><img src="images/adam.jpg" alt="Adam Conner-Simons">
```

Comment:
The speaker name classes ("adam", "aaron", etc.) are never used and could be removed:
```suggestion
                <figure><img src="images/adam.jpg" alt="Adam Conner-Simons">
```

Reply:
Also, while this `alt` text is serviceable, I'd also recommend thinking about what the photo communicates to a conference goer -- about if the speaker is serious or friendly, about the speaker's gender and ethnicity -- and try to make sure a potential attendee using a screenreader gets the same info.